calculator.js

```diff
@@ -1,28 +1,69 @@
+// const printInputs = function(num1, num2)
 
-const exampleAdditionInput = {
-  num1: 3,
-  num2: 5,
-  operation: 'add',
+const addCalc = function(num1, num2) {
+  result = (num1 * 1) + (num2 * 1)
+  console.log(`${num1} + ${num2} = ${result}`)
+};
```

Comment:
Here's a blueprint of how to tweak your helper methods below this one!
```suggestion
const addCalc = function(num1, num2) {
  const result = (num1 * 1) + (num2 * 1);
  console.log(`${num1} + ${num2} = ${result}`);
};
```